CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v1.1.3 (not published yet)
```

Comment:
I think we're moving to the habit of "new icons = minor version" to better align with semantic versioning. So this would be 1.2.0 instead.